lib/src/main/kotlin/com/swmansion/starknet/data/types/StarknetByteArray.kt

```diff
@@ -0,0 +1,27 @@
+package com.swmansion.starknet.data.types
+
+import com.swmansion.starknet.data.types.conversions.ConvertibleToCalldata
+import com.swmansion.starknet.extensions.splitToShortStrings
+import com.swmansion.starknet.extensions.toFelt
+
+data class StarknetByteArray(
```

Comment:
I'm not sure about the naming, but `ByteArray` is a reserved name 😞

Reply:
Yeah, a pretty common problem. StarknetByteArray seems reasonable :+1: